servicex/__init__.py

```diff
@@ -86,9 +86,17 @@ def create_app(test_config=None,
 
         if app.config['OBJECT_STORE_ENABLED']:
             if not provided_object_store:
+                if 'MINIO_SECURED' in app.config:
+                    if isinstance(app.config['MINIO_SECURED'], bool):
+                        use_https = app.config['MINIO_SECURED']
+                    else:
+                        use_https = strtobool(app.config['MINIO_SECURED'])
```

Comment:
Why is this needed? Shouldn't the Helm chart guarantee this is a boolean?

Reply:
I believe the Helm chart should guarantee this but I believe I ran into an error due to this so I added a check